src/reducers/index.js

```diff
@@ -5,11 +5,14 @@ import { combineReducers } from 'redux';
 import todos from './todos';
 import categories from './categories';
 import modal from './modal';
+import filter from './filter';
+import undoable from 'redux-undo';
```

Comment:
Implementing `undoable` by your own would be a good exercise. To learn and practice how to implement higher order reducers. I encourage you to try it, when you have some time ;)